staging/src/k8s.io/apiextensions-apiserver/test/integration/subresources_test.go

```diff
@@ -275,6 +304,7 @@ func TestScaleSubresource(t *testing.T) {
 	gottenScale.Spec.Replicas = 5
 	gottenScale.Status.Selector = "baz"
 	updatedScale, err := scaleClient.Scales("not-the-default").Update(groupResource, gottenScale)
+	// TODO(roycaihw): update this test to support multi-version.
```

Comment:
can we do this here? We are adding multi-version scale support. We should test it.

Reply:
I will update the test after https://github.com/kubernetes/kubernetes/issues/68197 is fixed.